### PR TITLE
Update CHANGELOG.md for v1.13.1

### PR DESCRIPTION
**Pull request by zmrow**
**Issue number:**
N/A

**Description of changes:**
Adds the CHANGELOG entry for `v1.13.1`

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by ecpullen — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]
LGTM!

**Review by webern — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# v1.13.1 (2022-03-21)
+
+## OS Changes
+
+* Ensure data partition can be grown to 20GiB on first boot ([#2920])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Enforce a minimum size of 20 GiB for the data partition ([#2920])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+### Kubernetes
+
+* Change CredentialProviderConfig api version to `v1beta1` for Kubernetes 1.25 variants ([#2906])
+
+## Platform Changes
+
+### AWS
+
+* Fix AWS profile rendering in credential provider ([#2904])
```
Comment (bcressey) [inline CHANGELOG.md:12]: This should go in the Kubernetes section since it applies to `*-k8s`.

**Comment by zmrow** [comment]
Addresses @bcressey 's comments

**Review by yeazelm — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**